pkg/skaffold/runner/runner.go

```diff
@@ -293,34 +293,53 @@ func (r *SkaffoldRunner) Run(ctx context.Context, out io.Writer, artifacts []*la
 	return nil
 }
 
+type tagErr struct {
+	tag string
+	err error
+}
+
 // imageTags generates tags for a list of artifacts
 func (r *SkaffoldRunner) imageTags(ctx context.Context, out io.Writer, artifacts []*latest.Artifact) (tag.ImageTags, error) {
 	start := time.Now()
 	color.Default.Fprintln(out, "Generating tags...")
 
-	tags := make(tag.ImageTags, len(artifacts))
+	tagErrs := make([]chan tagErr, len(artifacts))
+
+	for i := range artifacts {
+		tagErrs[i] = make(chan tagErr, 1)
+
+		i := i
```

Comment:
am I missing something here?

Reply:
The variable needs to be captured otherwise every go routine will use the same value

Reply:
makes sense. just looks kinda funny lol